test_data/makedata.js

```diff
@@ -16,6 +16,7 @@
 // `--progress [false]          whether to output a keepalive indicator to signal the invoker that work is ongoing
 // `--bigDoc                    Increase size of the graph documents
 // `--test                      comma separated list of testcases to filter for
+// `--tempDataDir               directory to store temporary data
```

Comment:
an ArangoDB PR would need to specify this in the rta-makedata suite, so the file can be cleaned up.

Reply:
Now temp files will not be created by default.